notification/k8s/values.yaml

```diff
@@ -0,0 +1,40 @@
+fullnameOverride: 'ctp-adyen-integration-service-notification'
+
+image:
+  repository: 'eu.gcr.io/professionalserviceslabs/commercetools-adyen-integration-notification'
```

Comment:
I'm not sure if we need to put all these CT-specific details here in the open-source repo. Especially if we don't have anywhere else `professionalserviceslabs`. Rather use docker hub image or sth fake.

Reply:
fe699ba853a0f361885cf7460e6ea2bc8ba7321b